### DISCUSSION TITLE
How to control sync timing

### DISCUSSION THREAD

**Discussion by VuDuc09** [HewlettPackard/swarm-learning#242] [Q&A]
I have a project with 3 hosts participating in the training process and there is a difference in data between the hosts (ratio 1:2:3). How can I control the sync process, specifically I want to perform a sync after each local epoch (whichever host finishes first will wait for the remaining hosts)

**Comment by PNg-HA** [comment]


**Reply by VuDuc09** [reply]
That's right, so that host will perform the sync without waiting for the remaining hosts to complete the local epoch
